examples/linux.rs

```diff
@@ -1,12 +1,12 @@
 //! Test driver with an SHTC3 sensor on Linux.
 
 use linux_embedded_hal::{Delay, I2cdev};
-use shtcx::{PowerMode, ShtCx};
+use shtcx::{self, PowerMode};
 
 fn main() {
     let dev = I2cdev::new("/dev/i2c-1").unwrap();
     let address = 0x70; // SHTC3
-    let mut sht = ShtCx::new(dev, address, Delay);
+    let mut sht = shtcx::shtc3(dev, address, Delay);
```

Comment:
Should the I2C address be hardcoded for the sensors? There would still be the generic variant, right?

Reply:
Hm, we could hardcode them, true. As long as we have the generic variant as an escape hatch.

Reply:
Done :)